### DISCUSSION TITLE
OPT-999 Dummy Tender Award Date

### DISCUSSION THREAD

**Discussion by Colan71** [OP-TED/eForms-SDK#284] [Q&A] [answered]
Hi,
How should we address OPT-999 Dummy Tender Award Date? Is that just a temporary OPT? In the XML examples, the Tender Award Date is placed on main level. Is that correct? Shouldn't the award date be on lot level since different lots can be awarded at different dates?

**Comment by YvesJo** [comment, marked as answer]
Hi,
OPT-999 Dummy Tender Award Date has been  introduced to satisfy UBL requirements, the associated element /*/cac:TenderResult/cbc:AwardDate is required in a Contract Award Notice. 
The information you are referring to is actually the Winner Decision Date (BT-1451, date of the official decision choosing the winning tender).

**Reply by Colan71** [reply]
Hi,
Can you please be a bit more precise about how we need to address OPT-999?

**Reply by YvesJo** [reply]
The UBL 2.3 standard requires the existence of /ContractAwardNotice/cac:TenderResult and /ContractAwardNotice/cac:TenderResult/cbc:AwardDate while none of these are associated to any Business information. OPT-999 does not have any meaning and has just been defined to help deal with the constraint imposed by the standard.
For any notice based on ContractAwardNotice, the following should be present:
```
	<cac:TenderResult>
		<cbc:AwardDate>2000-01-01Z</cbc:AwardDate>
	</cac:TenderResult>
```
Since OPT-999 does not have any meaning and is only there to satisfy the XSD, no other value than 2000-01-01Z should be used.

**Comment by Rubio1980** [comment]
Hi,

I'm sorry!

I don't understand.

Att.,


Em ter., 3 de jan. de 2023 às 11:40, Colan71 ***@***.***>
escreveu:
